### PR TITLE
Update .Net Url

### PR DESCRIPTION
**Pull request by MahmoudHamdy00**
As the previose was 404 not found

## What does this PR do?
Add resource(s) or just correct it for the same resource

### Description
 it's the same resource used for C# and it was already added but the url is wrong

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)
- [ ] Search for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by LuigiImVector — COMMENTED** [review]
I think that this resource should be moved to the C# category on the file [`free-programming-books-langs.md`](https://github.com/EbookFoundation/free-programming-books/blob/main/books/free-programming-books-langs.md)

**Comment by MahmoudHamdy00** [comment]
@LuigiImVector it already exists in C# category but on the same file

**Comment by LuigiImVector** [comment]
We are waiting for a response from some other staffer to figure out what to do. If you need to move it to the other file you also delete the link in the C# category of this file.

**Comment by davorpa** [comment]
@LuigiImVector comparing with archived version but it carry on having a handbook and interactive content so... I think leave it in the courses file. Moreover, the matrix `Epicodus` is like a learning platform.

LATEST BROKEN BACKUP: https://web.archive.org/web/20190115121159/https://www.learnhowtoprogram.com/net

I agree with move it to `C#` section and if this is the last remove `.NET` section and also from index

**Comment by eshellman** [comment]
This might be a good place to use the crosslink syntax #5535 that lets us remove the dead link and leave the heading.

**Comment by davorpa** [comment]
> This might be a good place to use the crosslink syntax #5535 that lets us remove the dead link and leave the heading.

Nice good point. Preparing suggestion :wink:

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -750,7 +750,7 @@
 
 ### .NET
```
Comment (davorpa) [inline courses/free-courses-en.md:752]:
```suggestion
### .NET

> :information_source: See also &#8230; [C#](#c-sharp)

```

It lefts moving record to `C Sharp` section (line 237) and adding below it heading...

```md
### C Sharp

> :information_source: See also &#8230; [.NET](#net)


````

Empty lines of above template are important

**Review by davorpa — CHANGES_REQUESTED** [review]

**Comment by eshellman** [comment]
Sorry if we were not clear; since it's a duplicate, the link to the course should be removed, and replaced with the crosslink to c#. Alternatively, if the course is more a .net course than a c# course, and use the updated link in .net and remove it from c# (adding a crosslink to c#)

File: courses/free-courses-en.md (hunk 1 above)
Comment (davorpa) [reply courses/free-courses-en.md:752]: @MahmoudHamdy00 not resolved yet

Crosslinks goes before lists (see for example Angular and Angular JS)

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -750,8 +750,9 @@
...
 
-* [Learn how to program: .NET](https://www.learnhowtoprogram.com/net) - Epicodus Inc.
+* [Learn how to program: C# and .NET](https://www.learnhowtoprogram.com/c-and-net) - Epicodus Inc.
```
Comment (davorpa) [inline courses/free-courses-en.md:753]: Removing duplicated link as Eric tell us

```suggestion
```

Is already present in C# section

**Review by davorpa — CHANGES_REQUESTED** [review]

File: courses/free-courses-en.md (hunk 2 above)
Comment (davorpa) [reply courses/free-courses-en.md]: Resolved by commit: [`b106347` (#6673)](https://github.com/EbookFoundation/free-programming-books/pull/6673/commits/b106347269925bfe72e559be653461e20b4ef1b8)

File: courses/free-courses-en.md (hunk 1 above)
Comment (davorpa) [reply courses/free-courses-en.md:752]: Resolved by commit: [`b106347` (#6673)](https://github.com/EbookFoundation/free-programming-books/pull/6673/commits/b106347269925bfe72e559be653461e20b4ef1b8)

**Review by davorpa — APPROVED** [review]
LGTM :wink:

**Pull request merged**